tools/ci_build/github/apple/build_ios_framework.py

```diff
@@ -102,6 +101,17 @@ def _build_for_ios_sysroot(
 
     return framework_dir
 
+def _merge_json_lists(files, output_file):
```

Comment:
can we call it `_merge_framework_info_files`?

Reply:
updated